src/screens/MainScreen/index.js

```diff
@@ -10,7 +11,10 @@ import styles from './styles';
 
 const MainScreen = () => {
   const dispatch = useDispatch();
-  const logoutRequest = useCallback(() => dispatch(logout()), [dispatch]);
+  const logoutRequest = useCallback(() => {
+    dispatch(logout());
+    LoginManager.logOut();
+  }, [dispatch]);
```

Comment:
Same here, you could have a `useLogout` hook

Reply:
Got it! Thank you!